templates/static/js/chart.js

```diff
@@ -0,0 +1,100 @@
+function responsivefy(svg) {
```

Comment:
Unfortunately it seems that adding these files in the same folder as example HTML is a bit of a problem.

There are two options here

- we keep them in `/static/js` (maybe just make it `static/js/examples` to know better what they are used for.
- or keep them with example HTML, but include them in HTML with `{% include %}` rather than `<script src="" />`, but this means that it will be inlined in HTML. it may be fine for short scripts, but we likely don't want whole d3 library inlined in the example

So, I guess keeping it in `static/js/examples` would be better for now, unless we have better ideas? @sowasred2012